code/style.css

```diff
@@ -1,7 +1,258 @@
+body {
+  margin: 0;
+  color: #fff;
+  background-color: #55739e;
+  font-family: 'Poppins', sans-serif;
+  width: auto;
+}
+
 /* the + sign is used to select the sibling that comes immediately next to the element you selected.
 in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp 
 
 .active + .answer {
-  /* some styling here */
+
+} */
+
+/* HERO SECTION */
+
+.hero-section {
+  position: relative;
+  height: 100vh;
+}
+
+.hero-video {
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+}
+
+.headline {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Work Sans', sans-serif;
+}
+
+.hero-title, .hero-tagline {
+  width: 80%;
+  text-align: center;
+}
+
+.hero-title {
+  line-height: 40px;
+  margin-bottom: 24px;
+  font-weight: bold;
+  font-size: 42px;
+}
+
+.hero-tagline {
+  margin-top: 0;
+  font-size: 34px;
+}
+
+/* CONTACT SECTION */
+
+.contact {
+
+  background-color: #494E61;
+  height: 600px;
+  width: 100%;
+}
+
+.contact-header {
+  margin-top: 50px;
+  font-size: 24px;
+}
+
+h2 {
+  margin-top: 0;
+  padding-top: 20px;
+  padding-left: 5px;
+}
+
+input { 
+  font-size: 16px;
+  border: none;
+  border-radius: 5px;
+  padding: 10px;
+  color: #55739e;
+}
+
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #fff;
+}
+
+.form-fields {
+  width: 80%;
+
+  
+}
+label {
+  display: flex;
+  flex-direction: column;
+  font-size: 20px;
+  width: 100%;
+  margin-top: 20px;
+}
+
+input:valid {
+  border-style: solid;
+  border-width: 5px;
+  border-color: #3C7A1B;
+}
+
+input:focus:required:invalid {
+  border-style: solid;
+  border-width: 5px;
+  border-color: #7A3022;
+}
```

Comment:
It might be confusing with the same styling for these

Reply:
Yes, I agree. The styling needs to be taken care of :)